tests/integration/filter-test.js

```diff
@@ -544,7 +554,13 @@ test('it is possible to filter created records by dirtiness', function(assert) {
   run(() => {
     customAdapter(env, DS.Adapter.extend({
       createRecord(type, model, snapshot) {
-        return Ember.merge(Ember.merge({}, snapshot._attributes), { id: snapshot.id} )
+        return {
+          data: {
+            id: snapshot.id,
+            type: model.modelName,
+            attributes: Ember.merge(Ember.merge({}, snapshot._attributes))
```

Comment:
Do we still need to specify `Ember.merge` here twice if we aren't merging the `id`?